doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,336 @@
+
+Storing keys in KMU
+===================
+
+Applications can store keys in KMU slots using the standard PSA cryptographic key management operations ``psa_import_key``, ``psa_generate_key`` or ``psa_copy_key``.
+Additionally, the KMU slots can be provisioned using the nRF Util development tool.
+
+.. note::
+   If a power failure occurs during provisioning of a key with persistence ``PSA_KEY_PERSISTENCE_READ_ONLY`` or ``CRACEN_KEY_PERSISTENCE_REVOKABLE``, it might not be possible to recover the key slot.
+   Provisioning of read-only keys should be restricted to controlled environments (production environments).
+
+You might encounter the following KMU-specific error codes when storing keys in KMU:
+
+* ``PSA_ERROR_ALREADY_EXIST``: One of the required key slots has already been provisioned.
+* ``PSA_ERROR_NOT_SUPPORTED``: Unsupported key type.
+
+Removing or revoking keys from KMU
+==================================
+
+Keys are deleted or revoked using the ``psa_destroy_key`` function.
+Calling the ``psa_destroy_key`` function on keys that have the persistence ``CRACEN_KEY_PERSISTENCE_REVOKABLE``, will mark the associated KMU slots as revoked, preventing the slots from being reused for new keys.
+
+
+Using KMU keys
+==============
+
+Keys stored in the KMU can be used in standard PSA cryptographic operations for encryption, decryption, signing a hash or a message, and verifying a hash or a message, given that the corresponding ``PSA_KEY_USAGE_*`` flags are set.
+
+Keys with the usage schemes Protected (``CRACEN_KMU_KEY_USAGE_SCHEME_PROTECTED``) and Seed (``CRACEN_KMU_KEY_USAGE_SCHEME_SEED``) can push data from the slots directly to CRACEN registers that are not accessible by the CPU.
+These usage schemes are supported for pushing symmetric keys used for cipher operations, and for seeds used by the CRACEN IKG.
+
+Keys with the usage scheme Raw (``CRACEN_KMU_KEY_USAGE_SCHEME_RAW``), such as symmetric keys, are temporarily pushed to a RAM location by the CRACEN driver, and then loaded by CRACEN into the asymmetric engine before running operations like sign or verify.
+
+Key slots with the usage scheme Encrypted (``CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED``) also have to be decrypted to a temporary push location in RAM before they are used by CRACEN, which is handled by the CRACEN driver.
+
+When the application is built with TF-M, this temporary push location is protected inside the secure processing environment to avoid exposing the key material to the non-secure application.
+If TF-M is not used, the keys are pushed to a reserved RAM area at location 0x20000000-0x20000064 (``kmu_push_area``).
+
+You might encounter the following KMU-specific error codes when using the KMU keys:
+
+* PSA_ERROR_INVALID_HANDLE: Attempting an operation on an empty KMU slot.
+* PSA_ERROR_NOT_PERMITTED: Attempting an operation on a revoked key.
+* PSA_ERROR_HARDWARE_FAILURE: The key slot has invalid data.
+* PSA_ERROR_CORRUPTION_DETECTED: The key slot has invalid data.
```

Comment:
```suggestion
* ``PSA_ERROR_INVALID_HANDLE``: Attempting an operation on an empty KMU slot.
* ``PSA_ERROR_NOT_PERMITTED``: Attempting an operation on a revoked key.
* ``PSA_ERROR_HARDWARE_FAILURE``: The key slot has invalid data.
* ``PSA_ERROR_CORRUPTION_DETECTED``: The key slot has invalid data.
```
See https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120291747/Typographical+conventions - Value

Reply:
Just a miss I guess. The ones mentioned earlier, under section Storing keys in KMU are ok. Will fix this.